docs/table.md

```diff
@@ -68,6 +69,7 @@ Function Name | Prop       | CSS Property    | Theme Field
 `gridColumnGap` | `gridColumnGap` | `grid-column-gap` | `space`
 `gridColumn` | `gridColumn` | `grid-column` | none
 `gridRow` | `gridRow` | `grid-row` | none
+`gridArea` | `gridArea` | `grid-area` | none
```

Comment:
I think you're missing `gridTemplateAreas` here.

Reply:
You are right, thanks for the catch!